### PR TITLE
Added Travis Build status to README.md

### PR DESCRIPTION
**Pull request by amitjamadagni**
Added Travis build status in README.md. Can we have this ?

### CODE REVIEW

**Comment by acroy** [comment]
Sure, why not.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
-# QuDynamics.jl
+QuDynamics.jl
+============
+
+[![Build Status](https://travis-ci.org/JuliaQuantum/QuDynamics.jl.png?branch=master)](https://travis-ci.org/JuliaQuantum/QuDynamics.jl)
```
Comment (acroy) [inline README.md:4]: Does one need `?branch=master`? I think for QuBase we just have 

```
[![Build Status](https://travis-ci.org/JuliaQuantum/QuBase.jl.svg)](https://travis-ci.org/JuliaQuantum/QuBase.jl)
```

Comment (amitjamadagni) [inline README.md:4]: I was going through two other packages and they seem to have it, may be its specific to master branch, I might be wrong but was in that thinking.

Comment (acroy) [inline README.md:4]: Yeah, seems you can specify for which branch you want to show the status. So I guess this is fine then.

**Comment by amitjamadagni** [comment]
Merging this!

**Pull request merged**